src/CalculatorWADTests/ScenarioStandard.cs

```diff
@@ -0,0 +1,160 @@
+//******************************************************************************
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using System.Threading;
+using System;
+
+namespace CalculatorTest
+{
+    [TestClass]
+    public class ScenarioStandard : CalculatorSession
+    {
+        private static WindowsElement header;
+        private static WindowsElement calculatorResult;
+
+        [TestMethod]
+        public void Addition()
+        {
+            Thread.Sleep(5000);
+            // Find the buttons by their names and click them in sequence to peform 1 + 7 = 8
+            session.FindElementByName("One").Click();
+            session.FindElementByName("Plus").Click();
+            session.FindElementByName("Seven").Click();
+            session.FindElementByName("Equals").Click();
+            Assert.AreEqual("8", GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void Division()
+        {
+            Thread.Sleep(5000);
+            // Find the buttons by their accessibility ids and click them in sequence to perform 88 / 11 = 8
+            session.FindElementByAccessibilityId("num8Button").Click();
```

Comment:
Let's use .FindElementByAccessibilityId for all these tests (or use the xpath style as in Subtraction(), if you want). We may not be able to make the tests entirely locale-independent, but we should do so where we can.

Reply:
Understood, updated other test cases to use Accessibility-ID.